doc/src/manual/documentation.md

```diff
@@ -337,11 +335,23 @@ y = MyType("y")
 ## Syntax Guide
 
 A comprehensive overview of all documentable Julia syntax.
-
 In the following examples `"..."` is used to illustrate an arbitrary docstring.
 
-`doc""` should only be used when the docstring contains `$` or `\` characters that should not
-be parsed by Julia such as LaTeX syntax or Julia source code examples containing interpolation.
+### `$` and `\` characters
+
+The `$` and `\` characters are still parsed as string interpolation or start of an escape sequence
+in docstrings too. The `raw""` string macro together with the `@doc` macro can be used to avoid
+having to escape them. This is handy when the docstrings include LaTeX or Julia source code examples
+containing interpolation:
+
+````julia
```

Comment:
Could it be that there is one superfluous backtick?

Reply:
No, the block would then be closed by the wrong triple backtick below.